test/app.js

```
@@ -0,0 +1,134 @@
+var mysql       = require('mysql');
+var moment      = require('moment');
+var connection = mysql.createConnection({
+        host     : 'host_IP',
+        user     : '-',
+        password : '-',
+        database : '-'
+});
+
+process.env['NODE_TLS_REJECT_UNAUTHORIZED'] = '0';
+var redis = require("redis"),
+        client = redis.createClient('6379','host-IP');
```

ESLint

[error] Combine this with the previous 'var' statement.

(one-var)